app/models/clip/text_embedding.rb

```diff
@@ -0,0 +1,19 @@
+class CLIP::TextEmbedding
+  def initialize(
+    tokenizer: Tokenizers.from_pretrained("M-CLIP/XLM-Roberta-Large-Vit-B-32"),
+    model_path: "./multilingual_model/textual.onnx"
+  )
+    @tokenizer = tokenizer
+    @model = OnnxRuntime::Model.new(model_path)
+  end
+
+  def call(text)
+    tensor = tokenizer.encode(text).ids
+    attention_mask = Array.new(tensor.size, 1)
```

Comment:
In our case we can just pass ids and attention mask of all ones, but normally you want a fixed size vector (ie. context window) and an attention mask that has ones in places with real ids and zeroes for places with padding tokens.